contents/founders/first-1000-users.md

```diff
@@ -0,0 +1,227 @@
+
+![history](https://res.cloudinary.com/dmukukwp6/image/upload/first_six_months_0a5368c875.png)
+
+In early pricing discussions, I negotiated ad-hoc with customers, going from "will anyone pay anything?" to a consistently used internal doc, which I iterated until it felt consistently right. 
+
+We had two goals at this stage:
+
+1. To figure out if people would pay anything for what we’d built.
+2. Create a pricing model technical users like ourselves would appreciate, which meant it had to be self-serve and totally transparent. 
+
+After a few weeks of trial and error, we had a public pricing table and started letting people put their card details in. Every time we made it more self-serve, we got a higher conversion rate.
+
+### Launching PostHog Cloud
+
+Once we’d nailed pricing, we launched PostHog Cloud in April 2020, two months after our initial launch on Hacker News. Now, getting started meant signing up and adding our autocapture snippet to their product. It took minutes.
+
+We launched with a limited free tier (much less than the 1 million events we offer today) and a 30-day free trial. We also made it “no card required” at signup, which massively increased conversions.
+
+Adding a cloud product with proper pricing massively improved our growth. It made self-serving even easier and increased confidence in the product. Keeping it low friction was important, though. Had we demanded people put in a card from the start, I’m certain people would have been put off.
+
+We reached 1,000 users in May 2020, a few weeks after the launch of PostHog Cloud and four months from when we started building. We had our first ever 
+
+## 🎓 Things I learned from reaching 1,000 users 
+
+### 1. Deadlines are motivating
+
+Our one-month plan had a hard deadline built-in. It forced us to make decisions about what we did and didn’t care about, and gave us a reason to push hard. It’s easy to drift without one.
+
+### 2. Have a plan for finding your first users before you start building
+
+It doesn’t need to be a detailed plan, but you should know who your users will be and how to reach them. We had a clearer plan for PostHog than our previous ideas.
+
+### 3. Optimize for learning, not revenue (at the start)
+
+Going self-serve too soon will reduce your opportunities to learn. The time we spent onboarding users by hand was incredibly valuable. We still do this with new PostHog products, like [our data warehouse](https://posthog.com/docs/data-warehouse) that’s now in open beta.
+
+### 4. Repeat users are more important than volume. 
+
+If people are using what you’ve built over and over again, chances are you’re solving a real problem. You don’t need hundreds of users to validate an idea. Don’t invest in a big launch until you’re certain users will stick.
```

Comment:
```suggestion
If people are using what you’ve built over and over again, chances are you’re solving a real problem. You don’t need hundreds of users to validate an idea. Don’t invest in a big launch until you’re certain users will stick. You may just find you never even need a big launch if you do this well - because you'll just start getting lots of word of mouth growth.
```